### PR TITLE
Add get/categories endpoint

### PR DESCRIPTION
**Pull request by BrokenRecord85**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const app = require('../app');
+const { response } = require('../app');
+
+
+beforeEach( () => seed(testData));
+afterAll( ()=> db.end());
+
+
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js:12]: Would be good to add an error handling test for an incorrect path, eg. /api/banana.

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
Left a comment for you to add into tests. Add this and send another PR. Other than that. Looking great! 😄

File: __tests__/app.test.js (hunk 1 above)
Comment (BrokenRecord85) [reply __tests__/app.test.js:12]: ok

**Review by Augs0 — APPROVED** [review]
Approved :) one thing to note, the names of your controllers / models is very general - just 'games' but this will be split into further sections like categories, reviews, etc. Bear this in mind when naming your files going forward

**Review by Rayhaan2k9 — COMMENTED** [review]
All good :)

**Pull request merged**